### PR TITLE
notifications: patron url

### LINKED ISSUE #802: Link to the patron profile not adapted to the concerned instance in the notification message.
**Describe the issue**
In the notifications sent (overdue, recall, availability, due_soon), the link to the patron account is not adapted to the instance deployed. It is currently always a broken link: `https://ils.rero.ch/rbnj/patrons/profile`

**To Reproduce**
1. See notifications sent in Gmail

**Expected behavior**
The link should not be broken, whenever it is on `ilspilot.test`, `ilsdev.test` or `ils.test`

**Software version**
ilsdev + ilspilot
rero-ils version c236bc8
rero-ils-ui version 4fc4656d

### PR DESCRIPTION
**Pull request by rerowep**
* Corrects patron url in notifications. `RERO_ILS_APP_URL` is used to
  construct the patron url. closes #802
* Adds comand line interface for notifications. Process of notifications
  can now be started with `invenio run notifications process`.
* Uses standart JSONSCHEMAS_ENDPOINT = '/schemas'.
* Uses JSONSCHEMAS_REPLACE_REFS = True to resolve $ref before serving a
  schema.
* Deletes all unnecessary `$schema` in fixtures.

## Why are you opening this PR?

- https://tree.taiga.io/project/rero21-reroils/task/1537

## How to test?

- Run setup and verify the patron urls are correct in the send emails.

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Extracted translations?

### CODE REVIEW

File: rero_ils/modules/notifications/api.py [hunk 1]
```diff
@@ -161,10 +161,10 @@ def replace_pids_and_refs(self):
             # create a link to patron profile
             patron = Patron.get_record_by_pid(data['loan']['patron']['pid'])
             view_code = patron.get_organisation().get('code')
-            base_url = current_app.config.get('RERO_ILS_APP_BASE_URL')
-            url_api = '{base_url}/{view_code}/patrons/profile'
-            profile_url = url_api.format(
-                base_url=base_url, view_code=view_code)
+            profile_url = '{base_url}/{view_code}/patrons/profile'.format(
+                base_url=current_app.config.get('RERO_ILS_APP_URL'),
```
Comment (zannkukai) [inline rero_ils/modules/notifications/api.py:165]: Why not use `get_base_url()` here ?

Comment (rerowep) [reply rero_ils/modules/notifications/api.py:165]: It is not the same. Maybe I have not chosen the right name for the variable.

**Comment by rerowep** [comment]
> Good news to do this. Can we split the JSONSchema document into several files? Languages, etc.

Should be interesting to test.

File: rero_ils/modules/notifications/api.py [hunk 2]
```diff
@@ -329,6 +329,15 @@ def process_notifications(cls, verbose=False):
 
         return count
 
+    @classmethod
+    def queue_info(cls):
```
[the remark this answers is not in the document]
Comment (rerowep) [reply rero_ils/modules/notifications/api.py]: No, I have deleted it.

**Pull request merged**